www/js/app.js

```diff
@@ -1051,16 +1074,30 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
                     $("#searchingArticles").show();
                 };
             };
+
+            if(! isDirEntryExpectedToBeDisplayed(dirEntry)){
+                console.debug(dirEntry, " is not expected to be displayed. Early retrun in readArticle().")
```

Comment:
This debug line seems unnecessary to me, because `isDirEntryExpectedToBeDisplayed` already outputs a log.
Or maybe you could merge them?